amundsen_application/models/preview_data.py

```diff
@@ -24,6 +24,6 @@ def __init__(self, columns: List = [], data: List = [], error_text: str = '') ->
 
 
 class PreviewDataSchema(Schema):
-    columns = fields.Nested(ColumnItemSchema, many=True)
+    columns = fields.Nested(ColumnItemSchema, many=True, unknown=EXCLUDE)
```

Comment:
`PreviewTest.test_good_client_response fails` because it passes many unknown fields (see https://github.com/amundsen-io/amundsenfrontendlibrary/blob/c6321020813e17fffef33a8b9a03d307233082a4/tests/unit/base/test_superset_preview_client.py#L18). 

Is this a test problem or should this schema handle that?